data/test/macros/unit_setup.cfg

```diff
@@ -47,6 +47,36 @@ no#endarg
     [/{ABIL}]
 #enddef
 
+#define TEST_ABILITY_OPPONENT ABIL VALUE OTHER
+
+#arg ID
+1#endarg
+
+#arg SELF
+no#endarg
+
+#arg ALLIES
+no#endarg
+
+#arg ENEMIES
+no#endarg
+
+#arg CUMULATIVE
+no#endarg
+
+    [{ABIL}]
+        id = {ID}
+        name = _"test-{ABIL}_{ID}"
+        affect_self = {SELF}
+        affect_allies = {ALLIES}
+        affect_enemies = {ENEMIES}
+        value = {VALUE}
+        cumulative = {CUMULATIVE}
+        apply_to=opponent
+        {OTHER}
+    [/{ABIL}]
+#enddef
```

Comment:
There's no need for a separate macro, just put `apply_to=opponent` in the `{OTHER}` argument of `TEST_ABILITY`.